### PR TITLE
Rl emission updates

### PR DESCRIPTION
**Pull request by RyanLucchese**
* changes coin emission from 10/20/30/40 to 10/10/40/40

### CODE REVIEW

**Review by levongh — DISMISSED** [review]

**Review by narek-h — DISMISSED** [review]

**Review by sedrakGR — DISMISSED** [review]

File: src/chainparams.cpp [hunk 1]
```diff
@@ -201,19 +201,19 @@ class CMainParams : public CChainParams {
         consensus.energiBackboneScript = CScript() << OP_HASH160 << ParseHex("b051bdceb44b28bb36ef2add5ec07ccbc64708c2") << OP_EQUAL;
 
         // Seeing as there are 526,000 blocks per year, and there is a 12M annual emission
-        // masternodes get 30% of all coins or 3.6M / 526,000 ~ 6.85
-        // miners get 20% of all coins or 2.4M / 526,000 ~ 4.57
-        // founder gets 10% of all coins or 1.2M / 526,000 ~ 2.28
+        // masternodes get 40% of all coins or 4.8M / 526,000 ~ 6.85
```
Comment (yghazi) [inline src/chainparams.cpp]: ~ 9.14

File: src/chainparams.cpp [hunk 2]
```diff
@@ -201,19 +201,19 @@ class CMainParams : public CChainParams {
...
+        // miners get 10% of all coins or 1.2M / 526,000 ~ 4.57
```
Comment (yghazi) [inline src/chainparams.cpp]: ~ 2.28

File: src/chainparams.cpp [hunk 3]
```diff
@@ -222,8 +222,8 @@ class CMainParams : public CChainParams {
         // which is around 4.8M / 26.07 ~ 184,000, where 26.07 are the
         // number of super blocks per year according to the 20160 block cycle
         consensus.nSuperblockCycle = 20160; // (60*24*14) Super block cycle for every 14 days (2 weeks)
-        consensus.nRegularTreasuryBudget = 18400000000000;
-        consensus.nSpecialTreasuryBudget = 400000000000000; // 4M special initial treasury budget
+        consensus.nRegularTreasuryBudget = 18400000000000ULL;
+        consensus.nSpecialTreasuryBudget = 418400000000000ULL; // 4M special treasury budget boost
```
Comment (yghazi) [inline src/chainparams.cpp]: I don't understand the `418400000000000`? Is it supposed to be 4 times the regular treasury budget?

File: src/chainparams.cpp (hunk 1 above)
Comment (RyanLucchese) [reply src/chainparams.cpp]: Thank you!

File: src/chainparams.cpp (hunk 3 above)
Comment (RyanLucchese) [reply src/chainparams.cpp]: I have re-expressed this in a way that hopefully makes more sense as:
```
consensus.nSpecialTreasuryBudget = 400000000000000ULL + consensus.nRegularTreasuryBudget; // 4 million extra coins for the special budget cycle
```
The idea being, the 4 million NRG for the special budget cycle represents a "boost", so the special budget cycle has a value of regular budget cycle + 4 million coins

Comment (yghazi) [reply src/chainparams.cpp]: Thank you for the clarification! :+1:

File: src/chainparams.cpp [hunk 4]
```diff
@@ -201,8 +201,8 @@ class CMainParams : public CChainParams {
         consensus.energiBackboneScript = CScript() << OP_HASH160 << ParseHex("b051bdceb44b28bb36ef2add5ec07ccbc64708c2") << OP_EQUAL;
 
         // Seeing as there are 526,000 blocks per year, and there is a 12M annual emission
-        // masternodes get 40% of all coins or 4.8M / 526,000 ~ 6.85
-        // miners get 10% of all coins or 1.2M / 526,000 ~ 4.57
+        // masternodes get 40% of all coins or 4.8M / 526,000 ~ 9.14
```
Comment (narek-h) [inline src/chainparams.cpp]: it's most probably intended, just to get clear understanding.  4.8M/526,000 = 9.125... 
Why do we approximate to 9.14 ?

**Review by narek-h — COMMENTED** [review]
looks good to me.

File: src/chainparams.cpp (hunk 4 above)
Comment (yghazi) [reply src/chainparams.cpp]: Cause we want all these individual subsidies to add up to the block subsidy (13.7), as the assertion states [here](https://github.com/RyanLucchese/energi/blob/e5fca1edcaa4fd3b287775bac77d8ff05c341351/src/chainparams.cpp#L219): 
```
        assert(consensus.nBlockSubsidyBackbone + consensus.nBlockSubsidyMiners + consensus.nBlockSubsidyMasternodes == consensus.nBlockSubsidy);
```

Comment (narek-h) [reply src/chainparams.cpp]: thanks @yghazi , got it.

**Review by narek-h — APPROVED** [review]

**Pull request merged**